Pool/PvP.vb

```diff
@@ -552,32 +584,54 @@ Public Class PvP
     End Sub
 
     Private Sub cbGames_SelectedIndexChanged(sender As Object, e As EventArgs) Handles cbGames.SelectedIndexChanged
+        ResetWinsAgainst()
+        If CurrentScreen = AppState.Winner Then
+            CurrentScreen = AppState.SelectPlayer
+        End If
         For Each gameEntry As DictionaryEntry In gamesHT
             If gameEntry.Value.Equals(cbGames.SelectedItem) Then
                 game = gameEntry.Key
                 Exit For
             End If
         Next
         allWins = player1.GetAllResults($"exec [selAllWins_v1] @gID={game},@output=0")
-        If allWins IsNot Nothing AndAlso allWins.Tables(0).Rows.Count > 0 AndAlso Not cbGames.SelectedItem.Equals("Choose") Then
+
+        If allWins IsNot Nothing AndAlso allWins.Tables(0).Rows.Count > 0 AndAlso ScoreTheme.ValidateCBox(cbGames) Then
             GetHighScores()
             lstAllWins.Visible = True
             lblScoreBoard.Visible = True
             btnSave.Enabled = True
```

Comment:
See if this can be moved in the permissions case. This button won't be visible or enable if they are a guest.

Reply:
Moved below to the permissions case.